backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtInput.java

```diff
@@ -53,14 +50,35 @@
 	boolean[] justPressedKeys = new boolean[256];
 	boolean[] justPressedButtons = new boolean[5];
 	InputProcessor processor;
-	char lastKeyCharPressed;
-	float keyRepeatTimer;
 	long currentEventTimeStamp;
 	final CanvasElement canvas;
+	final GwtApplicationConfiguration config;
 	boolean hasFocus = true;
+	GwtAccelerometer accelerometer;
 
-	public GwtInput (CanvasElement canvas) {
+	public GwtInput (CanvasElement canvas, GwtApplicationConfiguration config) {
 		this.canvas = canvas;
+		this.config = config;
+		if (config.useAccelerometer) {
+			if (GwtApplication.agentInfo().isFirefox())
+				setupAccelerometer();
+			else

```

Comment:
Could you add braces {} for the if and else clause?

Reply:
In https://github.com/libgdx/libgdx/wiki/Contributing#code-style it says 

> Curly braces may be omitted if it does not hurt readability.

so I think it's fine

Reply:
Well, in my honest opinion, I think it does hurt readability.
Omitting braces is fine if you have only one line in the if/else statement but you have 14 lines in the else statement.

If another developer changes something in Line 80 he might have forgotten that he is in an else-clause because the else-clause is far away.